src/docfx/config/ops/OpsMetadataRuleConverter.cs

```diff
@@ -13,6 +13,9 @@ internal static class OpsMetadataRuleConverter
     {
         public static string GenerateJsonSchema(string rulesContent, string allowlistsContent)
         {
+            Log.Write(rulesContent);
+            Log.Write(allowlistsContent);
```

Comment:
@928PJY , will this cause vscode extension to crash?

Reply:
no, vscode would not output Verbose logs

Reply:
@yufeih @OsmondJiang  For now, `--verbose` is not default enabled, but I am going to add a vscode config to make the user be able to enable that, but before that I will figure out a way to fix the crash caused by stdout size limitation, by make the limitation larger or flush the buffer when it catch some threshold.
anyway, for now, it is OK to add this log